src/constants/common.js

```diff
@@ -195,5 +195,4 @@ module.exports = {
 	RECOMMENDED_DURATION: 'recommended_duration',
 	NUMBER: 'number',
 	STRING: 'string',
-	SEQUELIZE_DATABASE_ERROR: 'SequelizeDatabaseError',
```

Comment:
Why was this removed here? @priyanka-TL . I think this was added as a bug fix right? is this change relevant here?